std/typecons.d

```diff
@@ -1759,7 +1759,16 @@ Practically $(D Nullable!T) stores a $(D T) and a $(D bool).
  */
 struct Nullable(T)
 {
-    private T _value;
+    /* T may be a struct with a @disabled this:
+       -If T supports default-initialization do so to allow use with @safe code
+       -If T does not support it then initialize with void
+    */
+    private enum defaultConstructed = __traits(compiles, {T _value;});
+    static if (defaultConstructed)
+        private T _value;
```

Comment:
This will achieve the same result as writing `T _value = T.init;` The only difference is that one of the compiles less often than the other. So the entire static if and condition is irrelevent. The only change here should be to add ` = T.init;`